metacat-common-server/src/main/java/com/netflix/metacat/common/server/usermetadata/BaseUserMetadataService.java

```diff
@@ -31,16 +33,12 @@
  */
 public abstract class BaseUserMetadataService implements UserMetadataService {
     /**
-     * Saves user metadata.
-     *
-     * @param userId user name
-     * @param holder metadata
-     * @param merge  true if the metadata should be merged with existing metadata
+     * {@inheritDoc}
      */
     @Override
-    public void saveMetadata(final String userId, final HasMetadata holder, final boolean merge) {
-        if (holder instanceof HasDefinitionMetadata) {
-            final HasDefinitionMetadata defDto = (HasDefinitionMetadata) holder;
+    public void saveMetadata(final String userId, final Metadata holder, final boolean merge) {
```

Comment:
Why did we rename HasMetadata and HasDefinitionMetadata? I would not recommend refactoring the names.